### PR TITLE
Update early disclosure list envoy requirements

### PR DESCRIPTION
**Pull request by howardjohn**
Based on feedback from the Envoy security team

A hold is applied until sufficient PSWG and Envoy security team review

### CODE REVIEW

**Review by kfaseela — DISMISSED** [review]

File: EARLY-DISCLOSURE.md [hunk 1]
```diff
@@ -62,16 +62,18 @@ email address, it must be a corporate address owned by the organization.
 3. Have a user base not limited to your own organization.
 4. Be a participant and active contributor in the Istio community.
 5. Accept the [Embargo Policy](#embargo-policy) that is outlined above.
+6. Have a member of your organization be a participant in the Istio Product Security Working Group. This is a private meeting held alternating Wednesdays at 11am Pacific or 3pm Pacific to accommodate a world-wide community.
```
Comment (justinpettit) [inline EARLY-DISCLOSURE.md:65]: Should we set out here the expectation that they will, on a quarterly basis, be expected to lead CVE releases during that time?

**Review by hzxuzhonghu — COMMENTED** [review]
LGTM

File: EARLY-DISCLOSURE.md [hunk 2]
```diff
@@ -113,10 +115,6 @@ in hacking the Gibson.
 
 We accept.
 
-> 6. Be a member of the [Envoy Early Disclosure list](https://github.com/envoyproxy/envoy/blob/main/SECURITY.md#members).
-
-We are listed in the Envoy early disclosure membership program.
-
 > 7. Have a member of your organization be a participant in the Istio Product Security Working Group
```
Comment (craigbox) [inline EARLY-DISCLOSURE.md:118]:
```suggestion
> 6. Have a member of your organization be a participant in the Istio Product Security Working Group
```

**Comment by justinpettit** [comment]
LGTM

**Review by keithmattix — APPROVED** [review]

**Review by alyssawilk — APPROVED** [review]
Looks great, thanks!

**Pull request merged**